modules/module-info.php

```diff
@@ -703,6 +703,11 @@ function jetpack_assetcdn_more_info() {
 /**
  * Copy Post
  */
+function jetpack_copy_post_more_link() {
+	echo 'https://jetpack.com/support/copy-post-2/';
```

Comment:
Looks like this should just be `https://jetpack.com/support/copy-post/`?

Reply:
I thought the only "Copy a Post" support article was for WordPress.com.

![screenshot_20190131-063414](https://user-images.githubusercontent.com/43215253/52035419-67acbb80-2522-11e9-9b82-ef56637dac88.jpg)

Reply:
I thought the only "Copy a Post" support article was for WordPress.com. That link just redirects me and causes an error.

![screenshot_20190131-063414](https://user-images.githubusercontent.com/43215253/52035488-9b87e100-2522-11e9-817d-384f1e5c6c46.jpg)

Reply:
I published the support doc to make testing easier:
https://jetpack.com/support/copy-post-2/

I unfortunately can't get the `copy-post` slug; for some reason it sticks to `copy-post-2`, I assume because there is already another post / term that uses that slug on the site. I could not find it, though, so I am not sure why that's happening s ¯\_(ツ)_/¯

Reply:
Yeah, weird. Looking at it yesterday, it was definitely just `/copy-post` for me, but looking now, I get the same as you, with nothing I can find holding on to the desired slug. Le sigh.

Reply:
It looks like a db error, like there was a `copy-post` _page_ (not support page) at some point – I also see weird redirecting to `/jetpack.com/copy-post`. Rather than risking making things worse by prodding the db, let's just go with this for now.